### ISSUE TITLE
Two meetings on the same day break the index

### ISSUE THREAD

**Issue by rhiaro** [w3c-ccg/meetings#41]
It looks like when there are two meetings on the same date, on the [meetings index page](https://w3c-ccg.github.io/meetings/) the second overrides the first in the list.

**Comment by TallTed** [comment]
I see two meetings with today's date (one with `-solidextra` suffix)... 

What's the breakage you're seeing?

**Comment by rhiaro** [comment]
I fixed that by hand

**Comment by vsnt** [comment]
Thanks for manually fixing the issue @rhiaro

**Comment by TallTed** [comment]
Maybe (big maybe) they should have full timestamps (for start-time, either in Z or W3-standard ET), rather than just datestamps... 

Maybe not, as multiple meetings on the same day will probably be as rare going forward as they are looking back.

Just thinking aloud.

**Comment by vsnt** [comment]
yeah @TallTed, I think the two meetings on the same day is not a normal use case, but it's good to document as an issue.
